testing/run-scripts/run_monitor.sh

```diff
@@ -0,0 +1,18 @@
+# Boot monitoring servers and metrics.
+
+if netstat -lnt | awk '$6 == "LISTEN" && $4 ~ ".8000"' >/dev/null; then
+  sh ./serve-zork-metrics.sh
+fi
+
+if ! docker ps -a | grep prom/prometheus >/dev/null; then
+  HOST_IP=`ip -o -4 addr list docker0 | awk '{print $4}' | cut -d/ -f1`
+  docker run -d -p 9090:9090 --add-host zork:$HOST_IP -v $PWD/prometheus.yml:/etc/prometheus/prometheus.yml prom/prometheus -config.file=/etc/prometheus/prometheus.yml -storage.local.path=/prometheus -storage.local.memory-chunks=10000
+fi
+
+if netstat -lnt | awk '$6 == "LISTEN" && $4 ~ ".9100"' >/dev/null; then
+  cd
```

Comment:
This will leave the file in the user's home directory. Much better to use `tmp` or, better yet, `mktemp` (see `image_make.sh`).